quickstart-node/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "monday-code-quickstart",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc",
+    "start": "node -r ts-node/register/transpile-only -r tsconfig-paths/register dist/index.js",
+    "dev": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "server": "nodemon",
+    "expose": "PORT=$(grep PORT .env | cut -d '=' -f2) && mapps tunnel:create -p ${PORT:-3000}",
+    "stop": "PORT=$(grep PORT .env | cut -d '=' -f2) && kill-port ${PORT:-3000}",
+    "fetch:schema": "bash fetch-schema.sh",
```

Comment:
these commands won't work on windows, which is the most popular OS that is being used by app developers

Reply:
True, I wanted to take the PORT variable from the .env file, but it is really hard to this for both windows and linux ssytems. So maybe i'll just use 3000 and the developer can change explicitly if he wants. What do you say?

Reply:
Make fetch:schema work in windows